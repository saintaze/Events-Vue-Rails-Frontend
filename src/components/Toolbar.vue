<template>
  <header class="header">
    {{brandName}}
  </header>
</template>

<script>
import { BRAND_NAME } from '@/constants.js';

export default {
  name: 'toolbar',
  data(){
    return {
      brandName: BRAND_NAME
    }
  }
}
</script>

<style lang="scss">
  .header {
    height: 40px;
    line-height: 40px;
    padding-left: 1.5em;
    background: #eee;
    border-bottom: 1px solid #ddd;
    font-family: Lucida Grande,Helvetica,Arial,Verdana,sans-serif;
    font-size: 14px;
  }
</style>