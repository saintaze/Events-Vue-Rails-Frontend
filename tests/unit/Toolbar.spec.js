import { shallowMount } from '@vue/test-utils';
import Toolbar from '@/components/Toolbar.vue'; 

let wrapper;

beforeEach(() => {
  wrapper = shallowMount(Toolbar, {
    propsData: {},
    mocks: {},
    stubs: {},
    methods: {},
  });
});

afterEach(() => {
  wrapper.destroy();
});

describe('Toolbar', () => {
  it('should be a Vue instance', () => {
    expect(wrapper.isVueInstance).toBeTruthy();
  });

  it('should render Html correctly', () => {
    console.log(wrapper.html())
    expect(wrapper.html()).toMatchSnapshot()
  })
});