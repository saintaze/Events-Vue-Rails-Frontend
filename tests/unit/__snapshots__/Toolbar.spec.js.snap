// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toolbar should render Html correctly 1`] = `
<header class="header">
  Event App
</header>
`;
